tools/torchci/td/get_reverts_caused_by_td.py

```diff
@@ -185,14 +185,70 @@ def get_td_exclusions(run_ids: tuple[int]) -> dict:
 @lru_cache
 def get_failures_additional_test_info(
     run_id: int,
-) -> dict[str, Any]:
+) -> list[dict[str, Any]]:
     """Fetches additional test info for failures in the given run_id."""
-    for i in range(3):
-        url = f"https://ossci-raw-job-status.s3.amazonaws.com/additional_info/reruns/{run_id}/{i + 1}"
-        response = requests.get(url)
-        if response.status_code == 200:
-            return response.json()
-    return {}
+
+    query = """
+    with job as (
```

Comment:
Nit: The query is not well formatted, maybe you can just throw it in CH console, format it, and copy the formatted query back here.